Nie-ontologies/Generic-ontologies/linguistic-morphology.ttl

```diff
@@ -201,28 +201,28 @@ lingmor:IndicativeMood
 lingmor:InjuncitveMood
 	rdfs:isDefinedBy <http://e-editiones.ch/ontology/linguistic-morphology#>;
 	a rdfs:Class;
-	rdfs:label "injunctive mood"@en, "Injunktiv"@de;
+	rdfs:label "injunctive mood"@en, "Injunktiv"@de;#tbd: injuctive = ? 
```

Comment:
I asked Reto, a computational linguist, who created this ontology, about this and the other issues.
"Modus" is generally not included in the German names.
'Injunktiv' is OK.